.github/workflows/publish-nym-vpn-app.yml

```
@@ -77,19 +68,20 @@ jobs:
     outputs:
       release_tag: ${{ steps.set_tag.outputs.tag }}
       release_id: ${{ steps.create_release.outputs.id }}
       version: ${{ steps.cargo-get.outputs.metadata }}
-      win_updater_exe: ${{ steps.win_updater.outputs.exe }}
-      win_updater_sig: ${{ steps.win_updater.outputs.sig }}
+      win_updater_x64_exe: ${{ steps.win_updater_x64.outputs.exe }}
+      win_updater_x64_sig: ${{ steps.win_updater_x64.outputs.sig }}
       win_updater_arm64_exe: ${{ steps.win_updater_arm64.outputs.exe }}
       win_updater_arm64_sig: ${{ steps.win_updater_arm64.outputs.sig }}
-      core_build_url_path: ${{ needs.build-windows.outputs.core_build_url_path }}
+      core_build_x64_url_path: ${{ needs.build-windows-x64.outputs.core_build_x64_url_path }}
```

actionlint (1.7.12)

[error] property "core_build_x64_url_path" is not defined in object type {core_build_url_path: string; upload_dir_windows: string}

(expression)